service/interpreter/workflowImpl.go

```diff
@@ -338,6 +340,11 @@ func executeState(
 	commandReqDone = true
 
 	commandRes := &iwfidl.CommandResults{}
```

Comment:
we also need to set the command result for startApiFailure: https://github.com/indeedeng/iwf-idl/blob/main/iwf.yaml#L843